### PR TITLE
Feature/update 2.0

### PR DESCRIPTION
**Pull request by ralphdoe**
Projects Fixed.
Jackson updated to 2.9.8.
Swagger-core pointing to 2.0.8-SNAPSHOT

### CODE REVIEW

File: java/inflector-dropwizard-guice/pom.xml [hunk 1]
```diff
@@ -30,8 +30,9 @@
 
         <dropwizard-version>1.3.1</dropwizard-version>
         <inflector-version>2.0.0-SNAPSHOT</inflector-version>
-        <jackson.version>2.9.5</jackson.version>
+        <jackson.version>2.9.8</jackson.version>
         <jersey2.version>2.23.1</jersey2.version>
+        <swagger-core-version>2.0.7-SNAPSHOT</swagger-core-version>
```
Comment (frantuma) [inline java/inflector-dropwizard-guice/pom.xml]: why is this 2.0.7? move property to root project and use 2.0.8-SNAPSHOT

File: java/inflector-jetty-webxml/pom.xml [hunk 2]
```diff
@@ -126,5 +131,6 @@
         <inflector-version>2.0.0-SNAPSHOT</inflector-version>
         <junit-version>4.8.2</junit-version>
         <slf4j-version>1.6.3</slf4j-version>
+        <swagger-core-version>2.0.8-SNAPSHOT</swagger-core-version>
```
Comment (frantuma) [inline java/inflector-jetty-webxml/pom.xml]: see comment above

File: java/inflector-dropwizard/pom.xml [hunk 3]
```diff
@@ -30,8 +30,9 @@
 
         <dropwizard-version>1.3.1</dropwizard-version>
         <inflector-version>2.0.0-SNAPSHOT</inflector-version>
-        <jackson.version>2.9.5</jackson.version>
+        <jackson.version>2.9.8</jackson.version>
         <jersey2.version>2.25.1</jersey2.version>
+        <swagger-core-version>2.0.8-SNAPSHOT</swagger-core-version>
```
Comment (frantuma) [inline java/inflector-dropwizard/pom.xml]: see comment above

File: java/java-jersey2-alternatives/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -88,13 +88,12 @@
     <servlet-name>OpenApi</servlet-name>
     <url-pattern>/openapi/*</url-pattern>
   </servlet-mapping>
-<!-- alternatively use own Bootstrap servlet
+
```
Comment (frantuma) [inline java/java-jersey2-alternatives/src/main/webapp/WEB-INF/web.xml]: why is this uncommented

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java [hunk 5]
```diff
@@ -19,7 +17,7 @@
                 version = "2.0",
                 description = "API Definition",
                 termsOfService = "Terms of service",
-                license = @License(name = "Apache 2.0", url = "http://foo.bar"),
+                license = @License(name = "Apache 2.0", url = ""),
```
Comment (frantuma) [inline java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java]: why the change to url?

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java [hunk 6]
```diff
@@ -31,15 +29,6 @@
         security = {
                 @SecurityRequirement(name = "req 1", scopes = {"a", "b"}),
                 @SecurityRequirement(name = "req 2", scopes = {"b", "c"})
-        },
```
Comment (frantuma) [inline java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java]: why are these removed?

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java [hunk 7]
```diff
@@ -31,15 +29,6 @@
...
-        servers = {
-                @Server(
-                        description = "server 1",
-                        url = "http://foo",
-                        variables = {
-                                @ServerVariable(name = "var1", description = "var 1", defaultValue = "1", allowableValues = {"1", "2"}),
-                                @ServerVariable(name = "var2", description = "var 2", defaultValue = "1", allowableValues = {"1", "2"})
-                        })
         }
```
Comment (frantuma) [inline java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java]: why are these removed?

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/operation/ServerOperationResource.java [hunk 8]
```diff
@@ -12,19 +12,12 @@
...
 )
 @Path("/operations")
 public class ServerOperationResource {
     @Path("/serversoperation")
     @GET
-    @Operation(operationId = "Pets", description = "Pets Example",
-            servers = {
-                    @Server(description = "server 2", url = "http://foo2")
-            }
-    )
```
Comment (frantuma) [inline java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/operation/ServerOperationResource.java]: why this class has been changed?

File: java/java-jersey2-alternatives/src/main/webapp/WEB-INF/web.xml (hunk 4 above)
Comment (ralphdoe) [reply java/java-jersey2-alternatives/src/main/webapp/WEB-INF/web.xml]: Fixed (commented again), doing some tests with the Bootstrap class.

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java (hunk 5 above)
Comment (ralphdoe) [reply java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java]: Changed by mistake. Fixed.

File: java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java (hunk 7 above)
Comment (ralphdoe) [reply java/java-jersey2-petstore/src/main/java/io/swagger/sample/petstore/openapidefintion/OpenAPIDefinitionResource.java]: Recovered, and added a new Server for testing .
